test/integration/crud/server_errors.test.js

```diff
@@ -86,30 +87,20 @@ describe('Errors', function () {
     'Cannot do exclusion on field b in inclusion projection'
   ]);
 
-  it('should return an error object with message when mixing included and excluded fields', {
-    metadata: { requires: { mongodb: '>3.0' } },
-    test: function (done) {
-      const db = client.db(this.configuration.db);
-      const c = db.collection('test_error_object_should_include_message');
-      c.insertOne({ a: 2, b: 5 }, { writeConcern: { w: 1 } }, err => {
-        expect(err).to.not.exist;
-        c.findOne({ a: 2 }, { projection: { a: 1, b: 0 } }, err => {
-          expect(PROJECTION_ERRORS).to.include(err.errmsg);
-          done();
-        });
-      });
-    }
+  it('should return an error object with message when mixing included and excluded fields', async () => {
+    const db = client.db(this.configuration.db);
```

Comment:
When using arrow functions in mocha we lose the proper `this` reference so `this.configuration` is undefined.

Reply:
done